main.go

```diff
@@ -0,0 +1,84 @@
+package main
+
+import (
+	"fmt"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/credentials/stscreds"
+	"github.com/aws/aws-sdk-go/service/s3"
+	"github.com/ministryofjustice/analytical-platform-auth0-user-exports/auth"
+	"log"
+	"os"
+	"time"
+
+	session2 "github.com/aws/aws-sdk-go/aws/session"
+	env "github.com/ministryofjustice/analytical-platform-auth0-user-exports/default_env_value"
+	connection "github.com/ministryofjustice/analytical-platform-auth0-user-exports/get_connection"
```

Comment:
same here, maybe call this package `github.com/ministryofjustice/analytical-platform-auth0-user-exports/connection`